diff -uNr breakin-2.2/Makefile breakin-2.2.installprefix/Makefile
--- breakin-2.2/Makefile	2011-12-31 22:53:59.094329736 -0500
+++ breakin-2.2.installprefix/Makefile	2012-01-03 23:59:39.981060586 -0500
@@ -3,7 +3,7 @@
 LDFLAGS= -DTHREADING 
 LIBS=-lpanel -lncurses `curl-config --libs`
 VERSION:=0.0
-INSTALLPREFIX=SET_ME
+INSTALLPREFIX=${DESTDIR}
 
 OBJS=breakin.o util.o dmidecode.o bench_stream.o bench_disk.o
 
